tests/test_testnet.py

```diff
@@ -316,23 +316,68 @@ def test_wait_for_transaction_timeout_and_expire():
         client.wait_for_transaction2(parent_vasp.account_address, 1, time.time() + 5, "hash", 0.1)
 
 
-def create_child_vasp_txn(parent_vasp, child_vasp):
+def test_get_parent_vasp_account():
+    client = testnet.create_client()
+    faucet = testnet.Faucet(client)
+
+    parent_vasp = faucet.gen_account()
+    child_vasp = LocalAccount.generate()
+    signed_txn = create_child_vasp_txn(parent_vasp, child_vasp)
+
+    client.submit(signed_txn)
+    client.wait_for_transaction(signed_txn)
+
+    account = client.get_parent_vasp_account(child_vasp.account_address)
+
+    expected_address = utils.account_address_hex(parent_vasp.account_address)
+    assert account.address == expected_address
+
+    account = client.get_parent_vasp_account(parent_vasp.account_address)
+    assert account.address == expected_address
+
+
+def test_get_parent_vasp_account_not_found():
+    client = testnet.create_client()
+    faucet = testnet.Faucet(client)
+
+    parent_vasp = LocalAccount.generate()
+
+    with pytest.raises(jsonrpc.AccountNotFoundError):
```

Comment:
This errors because just calling LocalAccount.generate() doesn't create the account onchain?

Reply:
yes